aicsimageio/readers/czi_reader.py

```diff
@@ -803,7 +803,10 @@ def physical_pixel_sizes(self) -> types.PhysicalPixelSizes:
             # so simply run array construct
             self.dask_data
 
-        return self._px_sizes  # type: ignore
+        if self._px_sizes is None:
+            raise ValueError("Pixel sizes weren't created as a part of image reading")
```

Comment:
why this change?

Reply:
Just to get rid of the `type: ignore`.

It is also "safer".

Pixel sizes are set during array construction but they _could_ error or they could still be None (in some very very very odd case). This basically solves the type error because are explicitly checking they were filled after invoking the array construction.